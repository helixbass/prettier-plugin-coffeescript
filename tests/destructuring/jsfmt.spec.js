run_spec(__dirname, ['coffeescript'])
