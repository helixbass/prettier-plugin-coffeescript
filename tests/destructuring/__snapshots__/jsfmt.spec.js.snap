// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`destructuring.coffee 1`] = `
[one, two = null, three = null] = arr
a = ([s=1,]) -> 1
{children, ...props} = this.props
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[one, two = null, three = null] = arr
a = ([s = 1]) -> 1
{ children, ...props } = this.props

`;
